packages/deployments/contracts/deploy/01_deployMessaging.ts

```diff
@@ -219,6 +219,7 @@ const handleDeploySpoke = async (
       !contract.includes("PolygonZk") &&
       !contract.includes("ZkSync") &&
       !contract.includes("Consensys") &&
+      !contract.includes("Axelar") &&
       !contract.includes("Multichain")) ||
```

Comment:
we can remove? or are we leaving the implementation in place.

imo we should delete